db/seeds.rb

```diff
@@ -7,3 +7,180 @@
 #   ["Action", "Comedy", "Drama", "Horror"].each do |genre_name|
 #     MovieGenre.find_or_create_by!(name: genre_name)
 #   end
+
+require 'net/http'
+require 'json'
+
+
+
+puts "----------------------------------------------------------------"
+puts "Records being destroyed"
+puts "----------------------------------------------------------------"
+
+
+
+puts "----------------------------------------------------------------"
+puts "Creating users..."
+puts "----------------------------------------------------------------"
+
+albert = User.create(
+  first_name: 'Albert',
+  last_name: 'Nikolli',
+  email: 'sarabanda.sarab@gmail.com',
+  password: 'azerty1234'
+)
+
+julien = User.create(
+  first_name: 'Julien',
+  last_name: 'Marchand',
+  email: 'liverpool.liver@gmail.com',
+  password: 'azerty1234'
+)
+
+ricardo = User.create(
+  first_name: 'Ricardo',
+  last_name: 'Canas',
+  email: 'benfica.benfi@gmail.com',
+  password: 'azerty1234'
+)
+
+puts "----------------------------------------------------------------"
+puts "#{User.count} Users created"
+puts "----------------------------------------------------------------"
+
+puts "----------------------------------------------------------------"
+puts "Creating players..."
+puts "----------------------------------------------------------------"
+
+#The code below is used to later change the nationality result in a compatible string format
+# string = "[\"Brazil\", \"Argentina\"]"
+# countries = JSON.parse(string).join(', ')
+# Player.all.each do |player|
+#   player.nationality = JSON.parse(player.nationality).join(', ')
+# end
+
+puts "----------------------------------------------------------------"
+puts "Creating players of liverpool"
+puts "----------------------------------------------------------------"
+
+url_liverpool = 'https://transfermarkt-api.vercel.app/clubs/31/players'
+uri = URI(url_liverpool)
+response = Net::HTTP.get(uri)
+data = JSON.parse(response)
+players = data['players']
+
+players.each do |player|
+  if player['marketValue'].present?
+    Player.create(
+    name: player['name'],
+    position: player['position'],
+    age: player['age'],
+    nationality: JSON.parse(player['nationality'].to_s).join(', '),
+    height: player['height'],
+    foot: player['foot'],
+    club: player['signedFrom'],
+    market_value: player['marketValue'].delete('^.0-9').to_i * 1000000,
+    price_per_day: player['marketValue'].delete('^.0-9').to_i * 1000000 / 365,
```

Comment:
On pourra peut-être demander à l'occasion s'il y a un moyen de refacto cette ligne car on ne tire pas l'info du price_per_day de l'API (méthode dans le modèle de base par exemple ?), mais c'est juste pour notre culture personnelle et notre pratique future.

Reply:
Which lines do you mean precisely??
We can also make some changes later on, for the moment if this is working well, we can populate the database.
I will go on and push this into the master.